src/components/Board/BoardShare/BoardShare.component.js

```diff
@@ -134,75 +137,77 @@ const BoardShare = ({
         </div>
         {isLogged && (
           <div className="ShareDialog__socialIcons">
-            <Button disabled={!isPublic} onClick={onCopyLink} color="primary">
-              <div className="ShareDialog__socialIcons__copyAction">
-                <div>
-                  <CopyIcon />
+            <PremiumFeature>
```

Comment:
The styles on labels are changend. Also on share you phrase share

Reply:
this issue is not related to these changes.